### PR TITLE
Revert MPES naming changes

### PR DESCRIPTION
**Pull request by lukaspie**
I made an own PR for these changes so that we can see that we are not breaking/missing anything. 

As [suggested ](https://github.com/FAIRmat-NFDI/nexus_definitions/pull/74#issuecomment-1994409787) by @rettigl, this reverts all name changes from mpes to photoemission, so that in the end, we have the old structure again:
```mermaid
graph TD
  A[NXmpes] --> B[NXxps]
  A[NXmpes] --> C[NXmpes_arpes]
  A[NXmpes] --> D[NXpeem]
  A[NXmpes] --> E[NXmpes_liquid]
```

The `energy` axis remains required as discussed. I have made all energy resolutions and calibrations optional, should we at least make the resolutions recommended if we require the `energy` axis?

### CODE REVIEW

**Comment by lukaspie** [comment]
@rettigl @domna please have a look here for the naming changes back to MPES instead of photoemission. Then we can finally merge #74 afterwards.

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 1]
```diff
@@ -1,68 +1,704 @@
...
+            * time-resolved angle-resolved X-ray photoelectron spectroscopy (trARPES)
+            * spin-resolved angle-resolved X-ray photoelectron spectroscopy (spin-ARPES)
+            * momentum microscopy
+        
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
+    geometries(NXcoordinate_system_set):
+      doc: |
+        Description of one or more coordinate systems that are specific to the setup
+        and the measurement geometry.
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:55]: this should be optional here, no?

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 2]
```diff
@@ -1,68 +1,704 @@
...
+          - |
+            Minimum distinguishable energy separation in the energy spectra.
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 10.24
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.24
+        resolution_errors(NX_FLOAT):
+          exists: optional
+          unit: NX_ENERGY
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:105]: I would remove this from the appdef.

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 3]
```diff
@@ -1,68 +1,704 @@
...
       (NXelectronanalyser):
+        device_information(NXfabrication):
+          exists: recommended
+          vendor:
+            exists: recommended
+          model:
+            exists: recommended
+          identifier:
+            exists: recommended
+        description:
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:194]: Should description really be required?

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 4]
```diff
@@ -1,68 +1,704 @@
...
+          doc: |
+            In case of a single or averaged gas pressure measurement, this is the scalar gas pressure around
+            the sample. It can also be an 1D array of measured pressures (without time stamps).
+        value_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_PRESSURE
+            doc: |
+              In the case of an experiment in which the gas pressure changes and is recorded,
+              this is an array of length m of gas pressures.
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:443]: I would also remove the log variants from the appdef

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 5]
```diff
@@ -1,68 +1,704 @@
...
+          doc: |
+            In case of a fixed or averaged electron current, this is the scalar current.
+            It can also be an 1D array of output current (without time stamps).
+        current_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_CURRENT
+            doc: |
+              In the case of an experiment in which the electron current is changed and
+              recorded with time stamps, this is an array of length m of current setpoints.
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:466]: similar

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 6]
```diff
@@ -1,68 +1,704 @@
...
+          experiment, including any activities performed on the individual instrument parts.
+          This group can be used to describe the preparation of the instrument prior to the
+          experiment, e.g. the cleaning procedure for a spin filter crystal.
+    (NXprocess_mpes):
+      exists: recommended
+      doc: |
+        Document an event of data processing, reconstruction, or analysis for this data.
+        The appropriate axis calibrations for a given experiment should be described using
+        one or more of the following NXcalibrations. The individual calibrations for a given
+        `AXISNAME` in `data` should be called `AXISNAME_calibration`.
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:480]: Should we still look into redesigning this? What was the outcome of our recent discussion?

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 7]
```diff
@@ -1,68 +1,704 @@
...
       energy_calibration(NXcalibration):
+        exists: optional
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:482]: I still don't like this very long list of options here in the appdef

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 8]
```diff
@@ -1,68 +1,704 @@
...
+      \@signal:
+        enumeration: [data]
+      data(NX_NUMBER):
+        unit: NX_ANY
+        doc: |
+          Represents a measure of one- or more-dimensional photoemission counts, where the
+          varied axis may be for example energy, momentum, spatial coordinate, pump-probe
+          delay, spin index, temperature, etc. The axes traces should be linked to the
+          actual encoder position in NXinstrument or calibrated axes in NXprocess.
+      energy(NX_NUMBER):
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:693]: This is defined in the NXmpes_data, no? Let's remove it here then.

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 9]
```diff
@@ -1,68 +1,704 @@
...
-                xref:
-                  spec: ISO 18115-1:2023
-                  term: 12.16
-                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
-      \@energy_indices:
-        exists: optional
+          exists: recommended
       \@energy_depends:
         type: NX_CHAR
         exists: optional
```
Comment (rettigl) [inline contributed_definitions/nyaml/NXmpes.yaml:704]: similar

**Review by rettigl — APPROVED** [review]
It looks mostly okay to me, but honestly is too large to completely review. 
If we merge this into the other branch, the comparison of this with the fairmat branch should be much smaller, as most changes are indeed already there, no?

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 1 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:55]: done

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 2 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:105]: removed

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 3 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:194]: made it recommended

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 4 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:443]: The disadvantage is that for pressure_gauge and flood_gun, the description of these logs is not defined anywhere else, so we would loose this information.

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 7 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:482]: Basically, we are just saying here that the `calibrated_axis` objects are recommended (and the same for some other sub-elements of the different calibration processes). We can also remove these `exists`keywords and then there is no need to have this long list.

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 8 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:693]: Removed, now it just says:
```
energy(NX_NUMBER):
  \@type:
    exists: recommended
```

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 9 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:704]: This is not defined in NXdata_mpes, so we would loose all the description in the docstring if we remove it here. We can also add the description of \@energy_depends to the base class if you prefer.

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 6 above)
Comment (lukaspie) [reply contributed_definitions/nyaml/NXmpes.yaml:480]: I am not sure. When I last brought up the topic in the TF meeting, I don't think we came up with something definitive. There is still some discussion [here](https://github.com/FAIRmat-NFDI/nexus_definitions/issues/181), but no conclusion so far.

**Pull request merged**